kernel/thread/thread.c

```diff
@@ -940,6 +943,19 @@ int thd_get_mode(void) {
     return thd_mode;
 }
 
+unsigned thd_get_hz(void) {
+    return thd_hz;
+}
+
+int thd_set_hz(unsigned hertz) {
+    if(!hertz || hertz >= 1000)
```

Comment:
```suggestion
    if(!hertz || hertz > 1000)
```
In the documentation you said 1-1000 are valid values

Reply:
Shit. You're right. I even fixed that one locally and forgot to commit. lol.